draft-ietf-quic-transport.md

```diff
@@ -3487,6 +3487,33 @@ This rule applies to all current and future QUIC frame types.  An endpoint
 MAY treat the receipt of a frame type that uses a longer encoding than
 necessary as a connection error of type PROTOCOL_VIOLATION.
 
+## Frames and Number Spaces {#frames_and_spaces}
```

Comment:
We use hyphens mostly.

```suggestion
## Frames and Number Spaces {#frames-and-spaces}
```

And up on line 3443 (where I can't comment) replace: "Section 4 of {{QUIC-TLS}}" with "{{frames-and-spaces}}".

Reply:
Give me couple hours, I will fix that after dinner.